test/conformance/kuttl/policy-validation/cluster-policy/success/policy-1-assert.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: vault-init-injector
+status:
+  conditions:
+  - reason: Succeeded
+    status: "True"
+    type: Ready
```

Comment:
What's our official stance on this? I thought we were moving away from asserting on the `conditions[]` array and instead looking for `status.ready` field.

Reply:
That's the opposite `status.ready` is deprecated.